website/app/styles/prism-dracula.scss

```diff
@@ -67,417 +67,453 @@
   --yellow-40: #f1fa8c66;
```

Comment:
what do you think of moving also the CSS variables (very generic names) under the `.doc-code-block__code-snippet` block, instead of having them under the `:root` ? in this way, everything related to the dracula theme is scoped.

Reply:
Sounds like a good idea, scoped them in https://github.com/hashicorp/design-system/pull/1749/commits/f4800b710381fc7db696f7f0aa1501b2c88c3f96